tasks/main.yml

```diff
@@ -32,15 +32,13 @@
 
 - name: install riak.conf with templated configuration
   template: "src={{ riak_conf_template }} dest=/etc/riak/riak.conf owner=root group=root mode=0444 backup=yes"
-  notify: restart riak
```

Comment:
if the notifies are here, it will call the handlers, which run after everything is done. if it was left, the new restart mechanism would trigger, and then this would trigger at the end of the run again.